### DISCUSSION TITLE
Possible to have UnoCSS enabled at the root, but uno.config.ts in subfolder (monorepo)?

### DISCUSSION THREAD

**Discussion by lockinmarv** [antfu/eslint-config#635] [Q&A]
Hey!

I have the whole `eslint.config.js` at the project root, with `unocss: true`. Inside `/apps/app` (Vite + React app) there is a `uno.config.ts` file and the unocss plugin installed in the Vite config. The eslint-plugin is also installed at the root.

In the terminal ESLint errors because it can't find the `uno.config.ts` in the root.

Is it possible that way or do I need to have a separate `eslint.config.js` in `/apps/app`, so it can find it from the current folder correctly? If so, how would the eslint config look like?

**Comment by ysansan98** [comment]
I've also encountered this problem. May I ask if you've solved it?

**Reply by lockinmarv** [reply]
I think I gave up and just used the one at the root, installing every needed eslint stuff there too (tailwind, React, Solid etc.) Maybe it's the cleanest solution

**Reply by ysansan98** [reply]
There is a solution: create a uno.config.ts file in the monorepo root directory with the following content:

```ts
export * from './packages/web/uno.config' 
```
